File: pset6/Mario2/mario.py
from cs50 import get_int

while True:
    answer = get_int("Height: ")
    if answer > 0 and answer < 9:
        break
for row in range(answer):
    for col in range(answer):
        calculate = answer - (row + 1)
        if col >= calculate:
            print("#", end="")
        else:
            print(" ", end="")
    print("  ", end="")
    for col in range(row + 1):
        print("#", end="")
    print()

File: pset1/Mario2/mario.c
#include <stdio.h>
#include <cs50.h>
int main(void)
{
    int n;
    do
    {
        n = get_int("Heigth: ");
    }
    while (n < 1 || n > 8);
    for (int i = 0; i < n; i++)
    {
        for (int j = 0; j < n; j++)
        {
            if (j >= n - (i + 1))
            {
                printf("#");
            }
            else
            {
                printf(" ");
            }
        }

        printf("  ");

        for (int t = 0; t <= i; t++)
        {
            // printf("#");
            if (t <= n + (i + 1))
            {
                printf("#");
            }
            else
            {
                printf(" ");
            }
        }
        printf("\n");


    }
}